src/main/java/domain/LottoMachine.java

```diff
@@ -0,0 +1,35 @@
+package domain;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class LottoMachine {
+
+    private final static int LOTTO_TICKET_PRICE = 1000;
```

Comment:
```suggestion
    private static final int LOTTO_TICKET_PRICE = 1000;
```
미세한 차이지만 요런 컨벤션이 있습니다ㅎㅎ;